src/backend/libs/efiling-account-client/src/main/java/ca/bc/gov/open/jag/efilingaccountclient/config/AutoConfiguration.java

```diff
@@ -1,19 +1,51 @@
 package ca.bc.gov.open.jag.efilingaccountclient.config;
 
-import ca.bc.gov.open.jag.efilingaccountclient.DemoAccountServiceImpl;
+import ca.bc.gov.ag.csows.accounts.AccountFacade;
+import ca.bc.gov.open.jag.efilingaccountclient.CsoAccountServiceImpl;
 import ca.bc.gov.open.jag.efilingaccountclient.EfilingAccountService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 import org.springframework.boot.autoconfigure.condition.ConditionalOnMissingBean;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
 
+import javax.xml.namespace.QName;
+import java.net.MalformedURLException;
+import java.net.URL;
+
 @Configuration
+@EnableConfigurationProperties(CsoAccountProperties.class)
 public class AutoConfiguration {
 
+    private static final Logger LOGGER = LoggerFactory.getLogger(AutoConfiguration.class);
+    private final CsoAccountProperties csoAccountProperties;
+
+    public AutoConfiguration(CsoAccountProperties csoAccountProperties) {
+
+        this.csoAccountProperties = csoAccountProperties;
+    }
+
+
+    public AccountFacade accountFacade() {
+
+        AccountFacade accountFacade = null;
+        try {
+
+            QName serviceName = new QName("http://accounts.csows.ag.gov.bc.ca/", "AccountFacade");
```

Comment:
dumb question: Are we ok with storing this URL in the repo

Reply:
Yup, these are wsdl namespaces from the wsdl file itself (which is also checked in).